tests/flow/test_graph_deletion.py

```diff
@@ -58,15 +57,47 @@ def populate_graph(cls):
         redis_graph.add_edge(edge)
         redis_graph.commit()
 
+    # Count how many nodes contains the `name` attribute
+    # remove the `name` attribute from some nodes
+    # make sure the count updates accordingly,
+    # restore `name` attribute from, verify that count returns to its original value.
+    def test01_delete_attribute(self):
+        # How many nodes contains the 'name' attribute
+        query = """MATCH (n) WHERE EXISTS(n.name)=true RETURN count(n)"""
+        actual_result = redis_graph.query(query)
+        nodeCount = int(actual_result.result_set[1][0])
+        assert(nodeCount == 7)
+
+        # Remove Tal's name attribute.
+        query = """MATCH (n) WHERE n.name = 'Tal' SET n.name = NULL"""
+        redis_graph.query(query)
+
+        # How many nodes contains the 'name' attribute,
+        # should reduce by 1 from previous count.
+        query = """MATCH (n) WHERE EXISTS(n.name)=true RETURN count(n)"""
+        actual_result = redis_graph.query(query)
+        nodeCount = int(actual_result.result_set[1][0])
+        assert(nodeCount == 6)
+
+        # Reintroduce Tal's name attribute.
+        query = """MATCH (n) WHERE EXISTS(n.name)=false SET n.name = 'Tal'"""
+        actual_result = redis_graph.query(query)
+
+        # How many nodes contains the 'name' attribute
+        query = """MATCH (n) WHERE EXISTS(n.name)=true RETURN count(n)"""
```

Comment:
It would be nice if we could use `EXISTS` as a unary filter (no `=true`), but we can push that off until after adding the new parser.